content/applications/hr/employees/new_employee.rst

```diff
@@ -231,36 +249,38 @@ This tab provides various fields for different information, depending on the cou
 located. Different fields are configured for different locations, however some sections appear
 regardless.
 
-- :guilabel:`Status`: If applicable, select a :guilabel:`Related User`, :guilabel:`Job Position`,
-  and :guilabel:`Language` with the drop-down menus. Type in the :guilabel:`Registration Number of
-  the Employee`, and the :guilabel:`NIF Country Code` if available.
-- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
-  Card` information here.
-- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
-  when the employee is working at a work center. This value affects the employee's pay, and may also
-  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
-  amount.
-- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
-  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
-  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
-  :guilabel:`Badge ID` to create a badge ID.
+- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
+  User`, with the drop-down menus.
+- :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in
```

Comment:
Consider breaking this bullet point up into sub-bullet points, especially the parts that are Belgium-specific.